biz.aQute.remote/src/aQute/remote/agent/RedirectOutput.java

```diff
@@ -132,4 +135,17 @@ public PrintStream getOut() {
 		return out;
 	}
 
+	public String getLastOutput() {
+		return lastOutput;
+	}
+
+	public void setLastOutput(String out) {
+		if (!"".equals(out) && out != null) {
+			out = out.replaceAll("^>.*$", "");
+
+			if (!"".equals(out) && !out.startsWith("true")) {
```

Comment:
I think that we can go ahead and merge this and if we need improved implementation in the future, we can always improve it.